core/tests/protractor_utils/CollectionEditorPage.js

```diff
@@ -153,9 +152,11 @@ var CollectionEditorPage = function() {
 
   // Set collection category.
   this.setCategory = async function(category) {
-    await action.select2(
-      'Editor Category Drop Down', editorCategoryDropdown.first(),
-      category);
+    await action.click('Category filter', categoryFilterDropdown);
+    var dropdownOption = element(
+      by.cssContainingText('mat-option .mat-option-text', category));
```

Comment:
ditto

Reply:
not done

Reply:
Can't be done due to reasons specified here. https://github.com/oppia/oppia/pull/12563#discussion_r697971790